contracts/modules/Checkpoint/DividendCheckpoint.sol

```diff
@@ -35,33 +35,19 @@ contract DividendCheckpoint is DividendCheckpointStorage, ICheckpoint, Module {
         _;
     }
 
-    /**
-     * @notice Function used to intialize the contract variables
-     * @param _wallet Ethereum account address to receive reclaimed dividends and tax
-     */
-    function configure(
-        address payable _wallet
-    ) public onlyFactory {
-        _setWallet(_wallet);
-    }
-
     /**
     * @notice Init function i.e generalise function to maintain the structure of the module contract
     * @return bytes4
     */
     function getInitFunction() public pure returns(bytes4) {
-        return this.configure.selector;
+        return bytes4(0);
     }
 
     /**
      * @notice Function used to change wallet address
      * @param _wallet Ethereum account address to receive reclaimed dividends and tax
      */
     function changeWallet(address payable _wallet) external onlyOwner {
-        _setWallet(_wallet);
-    }
-
-    function _setWallet(address payable _wallet) internal {
         require(_wallet != address(0));
```

Comment:
Wondering whether we should allow setting of address(0) now which would go back to using the dataStore default?